src/chapters/inheritance/exercises.rst

```diff
@@ -0,0 +1,11 @@
+Exercises: Inheritance
+======================
+
+Work on these exercises in the IntelliJ ``java-web-dev-exercises`` project.
+
+1. **Class design:** You are given a ``Computer`` class and need to add two additional classes: ``Laptop`` and ``SmartPhone``.
+   Before you start coding these classes, diagram how the three classes are going to be related to each other. 
+   Remember to add properties and methods to your diagram!
+2.  **Class implementation:** Implement your design and test it in a ``Program.java`` class.
```

Comment:
since you give them the files, id say the program.java rather than a program.java

Reply:
also, what do you think about adding a sub-bullet here w/ a note to implement 3 tests per class or something?

Reply:
Done! Thanks!